src/storage/item.go

```diff
@@ -62,11 +62,16 @@ type ItemFilter struct {
 	Status   *ItemStatus
 	Search   *string
 	After    *int64
+	IDs      *[]int64
+	SinceID  *int64
+	MaxID    *int64
 }
 
 type MarkFilter struct {
 	FolderID *int64
 	FeedID   *int64
+
+	Before *time.Time
```

Comment:
The value is declared, but never used. The marking implementation is incomplete.

Reply:
fixed